components/sup/src/http_gateway.rs

```diff
@@ -244,6 +259,32 @@ fn health(req: &mut Request) -> IronResult<Response> {
     }
 }
 
+fn service(req: &mut Request) -> IronResult<Response> {
+    let state = req.get::<persistent::Read<ManagerFs>>().unwrap();
```

Comment:
The bulk of code in the service and config functions is the same.. maybe add a helper function with the common code?